apps/server/src/infra/courses-client/courses-client.config.ts

```diff
@@ -0,0 +1,3 @@
+export interface CoursesClientConfig {

```

Comment:
@Fshmit Actually I think the naming should be left as it is. It's just for the sake of consistency in the folder structure. Keep in mind, in the modules/common-cartridge there is a CommonCartridgeConfig (later it could cause people to question as to why the config of CC is not in the right place)

Reply:
It's in the courses-client module, renaming it would be confusing and break with the conventions we have in the project.